docs/getting-started.rst

```diff
@@ -124,3 +124,83 @@ Now, lets send a reply to this message by referencing its `message_id`::
 We should see `hello again` appear in our telnet client.
 
 Those are the basics for sending and receiving messages via junebug. Take a look at junebug's :ref:`HTTP API documentation <http-api>` to see how else one can interact with junebug, and junebug's :ref:`CLI <cli-reference>` and :ref:`config <config-reference>` references for more on how junebug can be configured.
+
+Infrastructure Diagram
+----------------------
+This diagram is an example configuration of how all the parts of Junebug fit
+together in a typical setup.
+
+.. blockdiag::
+
+    diagram {
+        orientation = portrait;
+        default_group_color = lightblue;
+
+        C1 [label="USSD Channel"];
+        C1S [label="USSD Channel Status"];
+        C1M [label="USSD Channel Messages & Events"];
+        C2 [label="SMS Channel"];
+        C2S [label="SMS Channel Status"];
+        C2M [label="SMS Channel Messages Endpoint"];
+        SMS [label="SMPP SMS line"];
+        USSD [label="Dmark USSD line"];
```

Comment:
Hmm. What about something like "USSD line over HTTP"? It feels like perhaps
mentioning "HTTP" is trying to cram too much information into a tiny label
though. Happy for you to pick a wording that you like.